nrf70_bm_lib/source/nrf70_bm_core.c

```diff
@@ -11,8 +11,8 @@
 #include <zephyr/random/random.h>
 #endif /* CONFIG_NRF70_RANDOM_MAC_ADDRESS */
 
-#include "nrf70_bm_core.h"
 #include "nrf70_bm_lib.h"
+#include "nrf70_bm_core.h"
```

Comment:
Was this swap of order intended, @krish2718 ?

Reply:
Yes, the core.h relies on a structure definition in lib.h